pyproject.toml

```diff
@@ -21,6 +21,9 @@ uvicorn = "^0.11.5"
 geopandas = "^0.7.0"
 pylint = "^2.5.3"
 alembic = "^1.4.2"
+pytz = "^2020.1"
```

Comment:
Can we please try not include pytz? I think everything can be done using the built in datatime.timezone

Reply:
I'm not sure why this is showing up as a change I made, but pytz is being used in a bunch of places (env_canada, spotwx, test_get_hourlies, etc.). Getting rid of pytz is a larger refactor job than the scope of this ticket

Reply:
Fair enough - I've gone ahead an done that re-factor on my branch - so we'll get rid of it over time.